app/views/people/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Task Master Details</h1>
+
+<h2>Name: <%= @name %> </h2>
```

Comment:
Since you have access to the `@person` variable from the controller, you can use this to retrieve the name, occupation and color rather than creating new instance variables.